### PR TITLE
update: add Kotlin plugin contributing info

### PR DESCRIPTION
**Pull request by koshachy**
* add info on the Kotlin plugin contributing process
* divide Kotlin compiler/stdlib and tooling
* clean up

### CODE REVIEW

**Review by yalishevant — APPROVED** [review]
I'm ok with Kotlin plugins part

**Review by p7nov — COMMENTED** [review]
Just a couple of comments

File: docs/topics/contribute.md [hunk 1]
```diff
@@ -10,21 +10,30 @@ You can help us improve Kotlin by [participating in Kotlin Early Access Preview
...
-If you want to contribute to the Kotlin compiler, standard library, or tooling, go to [GitHub](https://github.com/jetbrains/kotlin), check out the latest Kotlin version,
+If you want to contribute to the Kotlin compiler and standard library, go to [JetBrains/Kotlin GitHub](https://github.com/jetbrains/kotlin), check out the latest Kotlin version,
  and follow [the instructions on how to contribute](https://github.com/JetBrains/kotlin/blob/master/docs/contributing.md).
- 
+
 You can help us by completing [open tasks](https://youtrack.jetbrains.com/issues/KT?q=tag:%20%7BUp%20For%20Grabs%7D%20and%20State:%20Open). 
 Please keep an open line of communication with us because we may have questions and comments on your changes. Otherwise, we won't be able to incorporate your contributions.
 
-## Contribute to other libraries and create your own
+## Contribute to the Kotlin plugin
```
[the remark this answers is not in the document]
Comment (koshachy) [reply docs/topics/contribute.md:21]: There are a lot of Kotlin plugins: CocoaPods, Gradle...
I'd like to use Kotlin IDE plugin, WDYT?

[the remark this answers is not in the document]
Comment (p7nov) [reply docs/topics/contribute.md:21]: Yes, this sounds good.

File: docs/topics/contribute.md [hunk 2]
```diff
@@ -10,21 +10,30 @@ You can help us improve Kotlin by [participating in Kotlin Early Access Preview
...
+To contribute to the Kotlin plugin, clone the [IntelliJ IDEA repository](https://github.com/JetBrains/intellij-community/) and follow the [instructions on how to contribute](https://github.com/JetBrains/intellij-community/blob/master/plugins/kotlin/CONTRIBUTING.md).
+
+## Contribute to other Kotlin libraries and tools
 
 Besides the standard library that provides core capabilities, Kotlin has a number of additional (kotlinx) libraries that extend its functionality. 
 Each kotlinx library is developed in a separate repository, has its own versioning and release cycle.
 
-If you want to contribute to a kotlinx library, follow the contribution process described for each library, such as [kotlinx.coroutines](https://github.com/Kotlin/kotlinx.coroutines),
- [kotlinx.serialization](https://github.com/Kotlin/kotlinx.serialization/blob/master/CONTRIBUTING.md), and [ktor](https://github.com/ktorio/ktor/blob/master/CONTRIBUTING.md).
+If you want to contribute to a kotlinx library (like [kotlinx.coroutines](https://github.com/Kotlin/kotlinx.coroutines) or
```
[the remark this answers is not in the document]
Comment (p7nov) [reply docs/topics/contribute.md:32]: Found an explanation: http://www.differencebetween.net/language/grammar-language/difference-between-such-as-and-like/

File: docs/topics/contribute.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (koshachy) [reply docs/topics/contribute.md:21]: Because we use the same plugin both for IDEA and AS

**Review by p7nov — APPROVED** [review]

File: docs/topics/contribute.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (koshachy) [reply docs/topics/contribute.md]: Yeah, missed that.
Fixed.

**Pull request merged**